docs/pages/eas/environment-variables.mdx

```diff
@@ -0,0 +1,193 @@
+
+Because your EAS Build and EAS Workflow jobs run on a remote server, these **.env** files might not be available. For instance, **.env** files are excluded from your uploaded project if they are listed in **.gitignore** or not committed to your local version control system.
+Additionally, you may want to use environment variables outside of your JavaScript code to customize your app binary at build time, such as setting a bundle identifier or a private key for an error reporting service.
+To accomodate for those needs we have a separate and different (but compatible) mechanism for managing environment variables in EAS, which is focused on storing and managing environment variables on the EAS servers and synchronizing them for local development.
+
+## Project-wide environment variables
+
+Project-wide environment variables are environment variables that are specific to a single EAS project.
+You can view and manage them by navigating to [Environment Variables](https://expo.dev/accounts/[account]/projects/[project]/environment-variables) section of the project configuration menu.
+These environment variables are available to your build jobs, workflow jobs and updates for this project. They can also be pulled locally for development if their visibility setting allows it.
+
+## Account-wide environment variables
+
+Account-wide environment variables are environment variables that are available across all of your projects in your EAS account.
+You can view and manage them by navigating to [Environment Variables](https://expo.dev/accounts/[account]/settings/environment-variables) section of the account settings menu.
+They are available to your build jobs, workflow jobs and updates, together with project-wide variables for this project.
+You can pull them locally or read outside of EAS servers if their visibility setting allows it.
+
+## Environments
+
+Currently EAS supports three environments for environment variables: `development`, `preview` and `production`.
+Environments are independent sets of environment variables that can be used to customize your app in different contexts.
+For example, you might want to use different API keys for development and production, or different bundle identifiers for different App Store releases.
+Environments allows you to do so. Every EAS Build and EAS Workflow job runs using environment variables from one of the available environments.
+Environments can also be used for updates, allowing you to easily use the same set of environment variables as for your build jobs, when creating an update using `eas update` command with `--environment` flag.
+
+The environment variables defined on EAS servers can be assigned to mutiple environments and have the same value across all of them, or be created for a single environment, so they can have different values for each environment.
```

Comment:
```suggestion
Environment variables can be assigned to multiple environments and have the same value across all of them, or be created for a single environment, so that you can set a specific value for a single environment.
```